pyfolio/timeseries.py

```diff
@@ -208,11 +155,13 @@ def omega_ratio(returns, annual_return_threshhold=0.0):
     ----------
     returns : pd.Series
         Daily returns of the strategy, noncumulative.
-         - See full explanation in tears.create_full_tear_sheet.
+        - See full explanation in :func:`~pyfolio.timseries.cum_returns`.
     annual_return_threshold : float, optional
-        Threshold over which to consider positive vs negative
-        returns. For the ratio, it will be converted to a daily return
-        and compared to returns.
+        Minimum acceptance return of the investor. Threshold over which to
+        consider positive vs negative returns. It will be converted to a
+        value appropriate for the period of the returns. E.g. An annual minimum
```

Comment:
Do you think it's worth explaining this a little bit more? I didn't get how we go from 100 to 0.018